tests/cp2k/test.py

```diff
@@ -0,0 +1,14 @@
+
+import dpdata
+
+cp2k_output = dpdata.LabeledSystem('cp2k_output', fmt = 'cp2k/output')
```

Comment:
I suggest to write a [unittest](https://docs.python.org/3/library/unittest.html)-based test framework in this file.